language-design.md

```diff
@@ -17,7 +17,34 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+> We should not make the Java programming language a cathedral, but a plain
+> bazaar might be too loose. What we need is more like a shopping mall, where there are
+> not quite as many choices but most of the goods are well designed and sellers stand up
+> and back what they sell. [Steele, 1998]
+
+We both agree with this quote for the most part, but we are a bit concerned that sometimes maintainers would disappear. If languages lean slightly cathedral from a shopping mall model, then they can have patterns for dealing with this problem. Real-life shopping malls kick vendors out if they stop paying rent, but this doesn't work when "Public APIs, like diamonds, are forever" [Bloch, 2006].
```

Comment:
Doesn't Java solve this with deprecation? It seems like code that's unmaintained for long enough and doesn't "keep up with the times" Java just deprecates that code, encouraging people to not use it but leaving it around so it doesn't break people's projects. It then adds that class to the deprecated list [here](http://docs.oracle.com/javase/7/docs/api/deprecated-list.html). 
The only way I can see this concern causing problems is if a maintainer disappears for code that's actually **broken** - presumably this doesn't happen particularly often because the review process for official Java libraries is rigorous (I hope!). Even if the code is broken, however, deprecating it is still an acceptable way to deal with the issue - it's not Java's responsibility to ensure that all the third party libraries work.